portal/gil/templates/gil/base.html

```diff
@@ -295,17 +312,16 @@ <h4 class="box-modal__title">{{_("Session Timed Out")}}</h4><p class="box-modal_
     {% endif %}
 {% endblock %}
 {% block init_scripts %}
-    <script src="{{url_for('static', filename='js/vendor.js') }}"></script>
-    <script src="{{url_for('static', filename='js/gil.js') }}"></script>
-    <script src="{{ url_for('static', filename='js/tnth.js') }}"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/i18next/8.4.3/i18next.min.js"></script>
     <script src="https://unpkg.com/i18next-xhr-backend@1.4.2/i18nextXHRBackend.js"></script>
     <script src="https://unpkg.com/i18next-browser-languagedetector@2.0.0/i18nextBrowserLanguageDetector.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/i18next/8.4.3/i18next.min.js"></script>
+    <script src="{{url_for('static', filename='js/vendor.js') }}"></script>
     <script src="{{ url_for('static', filename='js/i18next-config.js') }}"></script>
+    <script src="{{url_for('static', filename='js/gil.js') }}" defer></script>
+    <script src="{{ url_for('static', filename='js/tnth.js') }}"></script>
     {% if config.PIWIK_SITEID %}
         <script type="text/javascript">
```

Comment:
Looks like we can use [a separate script for the tracking code](https://matomo.org/faq/general/faq_20904/), but would require loading `PIWIK_DOMAINS` from the backend. 

Something to keep in mind, but there are easier inline scripts to remove first :)